mediawikijs/get_user_watchlist_feed.js

```diff
@@ -0,0 +1,21 @@
+// This file is autogenerated. See modules.json and autogenerator.py for details
```

Comment:
This pull request is for `API:Alldeletedreves` so I'm not sure why this file is here?

Reply:
@brendajerop do you want me to remove the code of the file- mediawikijs/get_user_watchlist_feed.js from module.json?

Reply:
You don't have to remove it. Just delete this file from this pull request as this pull request deals with API:Alldeletedrevisions

Reply:
deleted this file.

Reply:
I can still see this file?